docs/_docs/templates.md

```diff
@@ -421,56 +421,164 @@ The default is `default`. They are as follows (with what they filter):
+
+### Passing parameters to includes
+
+You can also pass parameters to an include. For example, suppose you have a file called note.html in your \_includes folder that contains this formatting: 
+ 
 ```liquid
-{% raw %}{% include footer.html param="value" variable-param=page.variable %}{% endraw %}
+{% raw %}<div markdown="span" class="alert alert-info" role="alert">
+<i class="fa fa-info-circle"></i> <b>Note:</b> {{include.content}}
+</div>{% endraw %}
 ```
 
-These parameters are available via Liquid in the include:
+The {% raw %}`{{include.content}}`{% endraw %} is a parameter gets populated when you call the include and specify a value for that parameter, like this:
 
 ```liquid
-{% raw %}{{ include.param }}{% endraw %}
+{% raw %}{% include note.html content="This is my sample note." %} {% endraw %}
 ```
 
-#### Including files relative to another file
+The value of `content` (which is `This is my sample note`) will be inserted into the {% raw %}`{{include.content}}`{% endraw %} parameter.
 
-You can also choose to include file fragments relative to the current file:
+Passing parameters to includes is especially helpful when you want to hide away complex formatting from your Markdown content. 
+
+For example, images with figure captions often have complicated formatting that you might want to simplify through an include with parameters. Here's an example:
+
+```html 
+<figure><a href="http://jekyllrb.com"><img src="logo.png" 
+style="max-width: 200px;" alt="Jekyll logo" /><figcaption>
+This is the Jekyll logo</figcaption></figure>
+```
+
+You could templatize this content in your include and make each value available as a parameter, like this:
 
 ```liquid
-{% raw %}{% include_relative somedir/footer.html %}{% endraw %}
+{% raw %}<figure>{% if {{include.url}} %}<a href="{{include.url}}">
+{% endif %}<img src="{{include.file}}" {% if {{include.max-width}} 
+%} style="max-width: {{include.max-width}}" {% endif %} 
+alt="{{include.alt}}" />{% if {{include.url}} %}</a>{% endif %}
+{% if {{include.caption}} %}<figcaption>{{include.caption}}
+</figcaption>{% endif %}</figure>{% endraw %}
 ```
 
-You won't need to place your included content within the `_includes` directory. Instead,
-the inclusion is specifically relative to the file where the tag is being used. For example,
-if `_posts/2014-09-03-my-file.markdown` uses the `include_relative` tag, the included file
-must be within the `_posts` directory, or one of its subdirectories. You cannot include
-files in other locations.
+This include contains 5 parameters:
+    
+* `url`
+* `max-width`
+* `file`
+* `alt`
+* `caption`
 
-All the other capabilities of the `include` tag are available to the `include_relative` tag,
-such as using variables.
+To account for optional parameters, the include uses `if` tags with the parameters. For example, `{% raw %}if {{include.url}}{% endraw %}` will include the `url` only if the `url` parameter is specified in the include.
```

Comment:
i think the `if` tag is appropriate here, because we're talking about optional parameters with includes. 

I added a link to Liquid's default filter.